content/docs/distribution/inbound_returns.md

```diff
@@ -94,7 +94,7 @@ You can only delete Inbound Returns if they have a status of <code>NEW</code>.
 
 ![Inbound list: delete](/docs/distribution/images/ir_delete.gif)
 
-## Creating a new manual Inbound Return
+## Creating an Inbound Return
```

Comment:
this was because the heading says 'a new manual Inbound' and then talks about automatically created returns, with another heading for 'manual returns' later on.